clients/js/main.ts

```diff
@@ -565,12 +559,6 @@ function parseAddress(chain: ChainName, address: string): string {
     throw Error("SUI is not supported yet")
   } else if (chain === "aptos") {
     throw Error("APTOS is not supported yet")
-  } else if (chain === "arbitrum") {
-    throw Error("ARBITRUM is not supported yet")
-  } else if (chain === "optimism") {
-    throw Error("OPTIMISM is not supported yet")
-  } else if (chain === "gnosis") {
-    throw Error("GNOSIS is not supported yet")            
```

Comment:
Did you mean to remove these?

Reply:
Yes. It was a mistake to add them yesterday. Those chains are covered by the isEVMChain() check above.